code/modules/uplink/uplink_items.dm

```diff
@@ -1756,14 +1756,14 @@ datum/uplink_item/role_restricted/superior_honkrender
 	name = "Holocarp Parasites"
 	desc = "Fishsticks prepared through ritualistic means in honor of the god Carp-sie, capable of binding a holocarp \
 			to act as a servent and guardian to their host."
-	item = /obj/item/storage/box/syndie_kit/carpian
+	item = /obj/item/guardiancreator/carp/rare
```

Comment:
why is this `/rare`

WHY

Reply:
its the 15 points that hippiestation gets (the rare holoparasite injector) which only gives more points.

Reply:
First of all, that sentence doesn't even make sense.
Second of all, `/rare` let you choose abilities like timestop and absolution.